File: iOS/APIExample-OC/APIExample-OC/Common/KeyCenter.m
//
//  KeyCenter.m
//  APIExample
//
//  Created by zhaoyongqiang on 2023/7/11.
//

#import "KeyCenter.h"

static NSString * const APPID = <#YOUR APPID#>;
static NSString * const Certificate = <#YOUR Certificate#>;

@implementation KeyCenter

+ (nullable NSString *)AppId {
    return APPID;
}

+ (nullable NSString *)Certificate {
    return Certificate;
}

@end

File: iOS/APIExample-Audio/APIExample-Audio/Common/KeyCenter.swift
//
//  KeyCenter.swift
//  OpenLive
//
//  Created by GongYuhua on 6/25/16.
//  Copyright © 2016 Agora. All rights reserved.
//

struct KeyCenter {
    static let AppId: String = <#YOUR APPID#>

    // assign token and certificate to null if you have not enabled app certificate
    static let Certificate: String? = <#YOUR Certificate#>
    // assign token and certificate to null if you have not enabled app certificate or you have set the certificate above.
    static var Token: String? = nil
}
